x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -89,7 +89,7 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number | stri
     name: i18n.translate('xpack.observability.enableInfrastructureView', {
       defaultMessage: 'Infrastructure feature',
     }),
-    value: false,
+    value: true,
```

Comment:
Do we need to have a setting for the infra tab?

Reply:
I got the answer during on-weekly mtg: due to time constraint,  it's enabled for now and in a follow up we'll remove the setting